code/modules/mob/living/simple_animal/slaughter/slaughter.dm

```diff
@@ -0,0 +1,433 @@
+///The following mini-antag was coded by 	KorPhaeron   for TG Station
+//Credits: Concept: KorPhaeron, Sprite:Ausops, Sounds:Cuboos
+///Ported for paradise by Aurorablade/Fethas
+///Things added for port:
+//Various tweeks
+//Blood...
+/// Made the message tell you waht the demon is sinking into(gibs/blood etc)
+///Alot of blood...Alot..Of blood..
+
+//////////////////The Monster
+
+/mob/living/simple_animal/slaughter
+	name = "Slaughter Demon"
+	real_name = "Slaughter Demon"
+	desc = "You should run."
+	speak_emote = list("gurgles")
+	emote_hear = list("wails","screeches")
+	response_help  = "thinks better of touching"
+	response_disarm = "flails at"
+	response_harm   = "punches"
+	icon = 'icons/mob/mob.dmi'
+	icon_state = "daemon"
+	icon_living = "daemon"
+	speed = 0
+	a_intent = "harm"
+	stop_automated_movement = 1
+	status_flags = CANPUSH
+	attack_sound = 'sound/misc/demon_attack1.ogg'
+	min_oxy = 0
+	max_oxy = 0
```

Comment:
... did you test this? max_oxy is supposed to define how much oxygen they can breath without being harmed. It should be INF if you want them immune to any oxygen changes.

Same for the rest of these.